src/aat/resources/features/common/case/befta-jurisdiction2-case-creation/Befta_Jurisdiction2_Default_Full_Case_Creation_Data.td.json

```diff
@@ -0,0 +1,190 @@
+{
+  "_guid_": "Befta_Jurisdiction2_Default_Full_Case_Creation_Data",
+  "_extends_": "Befta_Jurisdiction2_Case_Creation_Base_Data",
+  "users": {
+    "invokingUser": {
+      "username": "befta.caseworker.2.solicitor.2@gmail.com",
+      "password": "Pa55word11"
+    }
+  },
+  "request": {
+    "pathVariables": {
+      "jid": "BEFTA_JURISDICTION_2",
+      "ctid": "BEFTA_CASETYPE_2_1"
+    },
+    "body": {
+      "data": {
+        "_extends_": "Befta_Jurisdiction2_Default_Full_Case_Data"
+      }
+    }
+  },
+  "expectedResponse": {
+    "responseCode": 201,
+    "responseMessage": "OK",
+    "headers": {
+      "_extends_": "Common_Response_Headers",
+      "Vary": "Accept-Encoding"
+    },
+    "body": {
+      "id": "[[ANY]]",
+      "jurisdiction": "BEFTA_JURISDICTION_2",
+      "state": "TODO",
+      "version": 0,
+      "case_type_id": "BEFTA_CASETYPE_2_1",
+      "created_date": "[[ANY]]",
+      "last_modified": "[[ANY]]",
+      "security_classification": "PUBLIC",
+      "case_data": {
+        "DocumentField4": {
```

Comment:
Can you please extend it from 
  "_extends_": "Befta_Jurisdiction2_Default_Full_Case_Data"